src/components/contact/Contact.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { translate, Interpolate } from 'react-i18next';
+
+import '../../i18n/i18n';
+import styles from './assets/contact.scss';
+import peopleSearch from './assets/people-search.svg';
+import SocialMedia from './SocialMedia';
+
+const Contact = ({ t }) => {
+  const EmailLink = <a target="_blank" href={`mailto:${process.env.REACT_APP_EMAIL}`} className={styles.email}>{process.env.REACT_APP_EMAIL}</a>;
+  const SlackOrg = <a href={process.env.REACT_APP_SLACK_URL} className={styles.slackOrg}>{t('slack-org')}</a>;
+
+  return (
+    <section className={styles.App__Contact}>
+      <div className={styles.App__LeftSection}>
+        <img src={peopleSearch} className={styles.App__poepleSearch} alt="people search logo" />
```

Comment:
`poeple` -> `people`

Reply:
It looks here that you are switch from Pascal Case to camel case. `App_LeftSection` and `App_peopleSearch`. Let's try and stay consistent.

Reply:
Well, I used Pascal case and once I pulled the upstream development branch, I saw all class name change to follow different convention. Changed it to snake case as mentioned in below comments